client/spec/desks_management_spec.js

```diff
@@ -0,0 +1,18 @@
+
+'use strict';
+
+var desks = require('./helpers/desks');
+
+describe('desks management', function () {
+
+    beforeEach(function() {
+        desks.openDesksSettings();
+    });
+
+    it('lists macros under the Macro tab for new desks', function () {
+        desks.openDesksSettings();
```

Comment:
This is not necessary as `desks.openDesksSettings();` is covered by `beforeEach()` already.